apps/src/craft/designer/game/GameController.js

```diff
@@ -1339,7 +1344,71 @@ class GameController {
     }
     // check the final state to see if its solved
     if (this.levelModel.isSolved()) {
-      this.endLevel(true);
+      const player = this.levelModel.player;
+      if (this.checkHouseBuiltEndAnimation()) {
```

Comment:
Ideally I think these cut scenes would be implemented in `simple/craft.js` and not inside the game engine.  But I'll probably punt on this until after I update the craft repo.